core/src/main/scala/org/graphframes/examples/Graphs.scala

```diff
@@ -37,7 +37,7 @@ class Graphs private[graphframes] () {
    */
   def empty[T: TypeTag]: GraphFrame = {
     val _spark = spark
-    import _spark.implicits._
+    import _spark.implicits.*
```

Comment:
Just curious - is there new tooling that is leading to the addition of these type hints and import changes?

Reply:
Scala 2.13 is compatible (in theory) with scala-3. There was already a question about support of scala-3, I see as a good way to slowly migrate all the scala-2 syntax to the corresponding scala-3 syntax, for example, `_` -> `*`